server/server.ts

```diff
@@ -4,17 +4,21 @@ import cookieParser from "cookie-parser";
 import db from "./models/index";
 import v1Routes from "./routes/v1";
 
-const app = express();
-const port = 8080;
+if (process.env.JWT_SECRET) {
```

Comment:
```suggestion
if (!process.env.JWT_SECRET) {
    console.error(...);
    return 1;
}
```
It's better to have the bad path in an if block

Reply:
I got an error when I tried to return here because it says I'm not inside a function block?